src/lib.rs

```diff
@@ -57,8 +57,9 @@ use nom::combinator::{fail, map, map_res, opt, peek, recognize, value};
 use nom::multi::separated_list1;
 use nom::IResult;
 use parsers::{alphanums, hyphenated_alphanums, unsigned};
+use semver::Op;
```

Comment:
I would prefer if we implemented this ourselves instead of depending on `semver`.

Reply:
Sounds good, do we want a feature `semver` which will provide `versioning::Op::into::<semver::Op>` for downstream smoothness?

Reply:
Never mind -- thought about it and I don't see a good reason to have it particularly, but it can always be added later :)

Reply:
Yup let's leave it out entirely.